### DISCUSSION TITLE
GitHub Security Lab: Security Contact Needed

### DISCUSSION THREAD

**Discussion by sylwia-budzynska** [hhyo/Archery#2024] [General]
Hello,

The [GitHub Security Lab team](https://securitylab.github.com/) has found a potential vulnerability in Archery. Please create a [Security Advisory](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/creating-a-security-advisory) and invite me (sylwia-budzynska) in to further disclose and discuss the vulnerability details and potential fix. Alternatively, please add a [Security Policy](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository) containing a security email address to send the details to.

If you prefer to contact us by email, please reach out to securitylab@github.com with reference to GHSL-2022-101_GHSL-2022-108.

Kind regards,
Sylwia Budzynska
GitHub Security Lab

**Comment by LeoQuote** [comment]
hmmm 看着不太对，可能是诈骗的，小心求证

**Comment by LeoQuote** [comment]
@hhyo 你看要不按她说的，这个方案：

Alternatively, please add a [Security Policy](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository) containing a security email address to send the details to.

这样安全报告可以邮件发给你

**Comment by sylwia-budzynska** [comment]
Hello,

I’m a GitHub employee - on my profile I have the GitHub “STAFF” badge, which is given only to GitHub employees:
![image](https://user-images.githubusercontent.com/102833689/211365070-5a7a1f3e-4138-4ac6-a29b-5eaa7cc523ba.png)

If you prefer, you could also enable private vulnerability reporting, which allows security researchers to contact your team directly in case of a vulnerability (available in the Security tab > Overview > Enable private vulnerability reporting). For more information about private vulnerability reporting, see [GitHub docs](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability).

**Reply by LeoQuote** [reply]
Thanks for your reply! I’m not able to see the badge on mobile device, I’m a contributor of this project but have no access to repository settings, @hhyo would do the suggested settings, thanks for your contribution!

**Reply by hhyo** [reply]
> Thanks for your reply! I’m not able to see the badge on mobile device, I’m a contributor of this project but have no access to repository settings, @hhyo would do the suggested settings, thanks for your contribution!

已启用，看了邮件确实有相关报告，被直接归档了🐟
